devp2p/p2p_protocol.py

```diff
@@ -164,7 +164,9 @@ def create(self, proto, reason=reason.client_quitting):
             assert self.reason_name(reason)
             log.debug('send_disconnect', peer=proto.peer, reason=self.reason_name(reason))
             proto.peer.report_error('sending disconnect %s' % self.reason_name(reason))
-            gevent.spawn_later(1., proto.peer.stop) #working?
+            # Doc says we should wait 2 second before we disconnect.
+            # That can block node from accepting new peers for 2 seconds.
+            gevent.spawn_later(0.5, proto.peer.stop)
```

Comment:
I intended to explain in the comment that 2 sec is too much. Perhaps could be better without any comment there.